wrappers/services.go

```diff
@@ -389,6 +389,9 @@ X-Snappy=yes
 ExecStart={{.App.LauncherCommand}}
 SyslogIdentifier={{.App.Snap.InstanceName}}.{{.App.Name}}
 Restart={{.Restart}}
+{{- if .App.RestartDelay}}
+RestartSec={{.App.RestartDelay.Seconds}}
```

Comment:
The snapd version of the variable name is so much better than the systemd version!